### DISCUSSION TITLE
Lecture-02, Velocity formula in polar coordinate has a problem

### DISCUSSION THREAD

**Discussion by Amir-M-Vahedi** [cooperrc/advanced-dynamics#19] [Logistics] [answered]
Hi everyone,

I think the velocity formula in polar coordinates has a problem.
The $\dot\theta$ should be multiplied by the whole parentheses. (In second line of the image)
Now it looks like below:
<img width="736" alt="image" src="https://github.com/cooperrc/advanced-dynamics/assets/115154998/6859e8bd-3ec1-42d5-a300-5d2ba0663bb5">
but in the last line, it's written in the correct form.

**Comment by safdarmt** [comment]
I agree

**Comment by Maid-Unknown** [comment]
I think you are right

**Comment by cooperrc** [comment, marked as answer]
nice catch

**Reply by Amir-M-Vahedi** [reply]
Thanks.
